falcon/util/uri.py

```diff
@@ -25,21 +25,19 @@
 """
 
 from falcon.constants import PYPY
+
 try:
     from falcon.cyutil.uri import (
         decode as _cy_decode,
-        parse_query_string as _cy_parse_query_string
+        parse_query_string as _cy_parse_query_string,
     )
 except ImportError:
     _cy_decode = None
     _cy_parse_query_string = None
 
 
 # NOTE(kgriffs): See also RFC 3986
-_UNRESERVED = ('ABCDEFGHIJKLMNOPQRSTUVWXYZ'
-               'abcdefghijklmnopqrstuvwxyz'
-               '0123456789'
-               '-._~')
+_UNRESERVED = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-._~'
```

Comment:
Does `black` insist on joining these, or is possible to leave the separation as-is?

Reply:
yes, since they fit in the same line it will do `'abcdefghijklmnopqrstuvwxyz' '0123456789' '-._~'` I joined them together.

we could mark the section as format off, or use a tuple and join them. But I think it's fine to have it as one line.

Also maybe we could use something linke `string.ascii_letters + string.digits + '-._~'` ?

Reply:
This one is not critical for me, I just thought that the previous indentation was intentional.

Reply:
> I just thought that the previous indentation was intentional.

that's most likely true, but I think it's still understandable that's ascii + digits + some special chars

Reply:
string.ascii_letters + string.digits + '-._~' sounds like a good way to make it a little more self-documenting, but not critical to change it.

Reply:
The only thing to consider is that with the concatenation form the resulting string may not be interned by python (iirc the string literal are).

If we want to change this a follow on pr is probably best